src/lib/geo-location.cjs

```diff
@@ -19,12 +19,18 @@ const REQUEST_TIMEOUT = 1e4
  * @property {object} subdivision
  * @property {string} subdivision.code
  * @property {string} subdivision.name
+ * @property {number} longitude
+ * @property {number} latitude
+ * @property {string} timezone
  */
 
 const mockLocation = {
   city: 'San Francisco',
   country: { code: 'US', name: 'United States' },
   subdivision: { code: 'CA', name: 'California' },
+  longitude: 0,
+  latitude: 0,
+  timezone: 'Mock Timezone',
```

Comment:
[pebble] Could I suggest using a real timezone (e.g. UTC) so that the mock data can be used in places that need a valid tz string.